tests/fido-lite.rs

```diff
@@ -3,6 +3,68 @@ use sshcerts::fido::*;
 
 use ring::digest;
 
+const YUBIKEY_5C_NFC_5_7_4_AUTH_DATA_ED25519: [u8; 225] = [
+    24, 215, 9, 61, 183, 202, 121, 173, 239, 111, 95, 74, 89, 136, 195, 47, 121, 236, 155, 71, 59,
+    195, 142, 93, 31, 100, 49, 19, 43, 73, 121, 223, 65, 0, 0, 0, 2, 215, 120, 30, 93, 227, 83, 70,
+    170, 175, 226, 60, 164, 159, 19, 51, 42, 0, 128, 181, 107, 62, 199, 160, 106, 19, 187, 158,
+    247, 13, 245, 95, 181, 81, 174, 117, 166, 255, 206, 83, 215, 54, 88, 235, 32, 84, 17, 216, 53,
+    163, 197, 202, 80, 151, 32, 102, 235, 151, 229, 233, 63, 180, 255, 99, 3, 47, 255, 129, 70,
+    102, 236, 113, 178, 86, 188, 10, 241, 36, 0, 94, 243, 4, 86, 140, 102, 215, 71, 56, 117, 185,
+    49, 112, 232, 209, 217, 134, 253, 37, 132, 139, 196, 39, 48, 173, 93, 159, 8, 72, 103, 244, 98,
+    143, 101, 110, 94, 81, 209, 67, 200, 139, 198, 13, 237, 110, 56, 38, 166, 196, 113, 169, 152,
+    205, 210, 90, 83, 208, 120, 167, 220, 22, 82, 69, 238, 129, 233, 100, 16, 164, 1, 1, 3, 39, 32,
+    6, 33, 88, 32, 193, 17, 95, 182, 218, 154, 215, 23, 121, 92, 9, 147, 81, 175, 117, 92, 164,
+    240, 117, 205, 156, 217, 189, 219, 213, 79, 90, 76, 184, 70, 71, 93,
+];
+
+const YUBIKEY_5C_NFC_5_7_4_AUTH_SIG_ED25519: [u8; 70] = [
+    48, 68, 2, 32, 68, 224, 82, 70, 239, 132, 78, 26, 46, 145, 7, 206, 120, 107, 237, 123, 178, 68,
+    57, 14, 199, 171, 239, 31, 4, 68, 187, 43, 84, 112, 227, 227, 2, 32, 7, 60, 129, 110, 141, 39,
+    253, 238, 233, 102, 212, 213, 86, 188, 206, 26, 201, 13, 127, 69, 103, 79, 200, 112, 198, 207,
+    80, 77, 55, 73, 110, 46
+];
+
+const YUBIKEY_5C_NFC_5_7_4_CHALLENGE_ED25519: [u8; 32] = [
```

Comment:
@obelisk For the other tests in the same file, the challenge is a SHA256 preimage, but not this one.

Reply:
So if you can manage to derive the preimage for this, I'll plop that in :P